### DISCUSSION TITLE
Account Security and Support

### DISCUSSION THREAD

**Discussion by xihn** [community/community#55522] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi, I opened a support ticket however I feel that support is clearly not reading my answers and just providing generic automated or scripted responses. My account was compromised / hacked and support just suspended the account and essentially said kick rocks. I have a DigitalOcean account with my payment info on it that I had connected to my GitHub account. whoever had my GitHub account racked up a bit of charges now. Is there any way to have a ticket re-opened or to elevate to someone a tad more competent or technical.

**Comment by RevertIT-alt** [comment]
Try to provide as much as possible info, including logs, ips, mails, etc.

It might take a while, but if they close the ticket with bot response, you should create a new one.

In past it took me 3 attempts to recover my account with support as my old email provider ceased to exist and couldn't remember the password.
